File: frontend/src/components/utils/NetworkEndPoints.js
const BASE_URL = "https://password-manager-sandeep03edu-backend.onrender.com";

const EndPoints = {
  // Authentication
  emailExist: "api/auth/emailExist",
  login: "api/auth/login",
  registerUser: "api/auth/register",
  updateUser: "api/auth/update",
  verifyAppPin: "api/auth/verifyAppPin",

  // Cards
  securePaginatedAllCard: "api/credentials/securePaginatedAllCards?page=",
  fetchCardDetails: "api/credentials/fetchCardDetails",
  addUpdateCard: "api/credentials/addUpdateCard",
  deleteCard: "api/credentials/deleteCardById",

  // Passwords
  securePaginatedAllPassword:
    "api/credentials/securePaginatedAllPasswords?page=",
  fetchPasswordDetails: "api/credentials/fetchPasswordDetails",
  addUpdatePassword: "api/credentials/addUpdatePassword",
  deletePassword: "api/credentials/deletePasswordById",

  // Credentials
  fetchCredentialCount: "api/credentials/fetchCredentialCount",
  fetchMonthlyCredentialsData: "api/credentials/fetchMonthlyCredentialsData",
  fetchRecentEncryptedCredentials:
    "api/credentials/fetchRecentEncryptedCredentials",
};

module.exports = { BASE_URL, EndPoints };

File: MultiplatformMobile/shared/src/commonMain/kotlin/com/sandeep03edu/passwordmanager/manager/utils/data/NetworkEndPoints.kt
package com.sandeep03edu.passwordmanager.manager.utils.data

class NetworkEndPoints {

    companion object {
        const val loginUser : String = "/api/auth/login"
        const val registerUser : String = "/api/auth/register"
        const val updateUser : String = "/api/auth/update"

        const val restartApi : String = "/api/auth/restart"

        const val addUpdateCard: String =  "/api/credentials/addUpdateCard"
        const val addUpdatePassword: String =  "/api/credentials/addUpdatePassword"
        const val deleteCard: String =  "/api/credentials/deleteCardById"
        const val deletePassword: String =  "/api/credentials/deletePasswordById"
    }
}